mediaapi/routing/routing.go

```diff
@@ -83,26 +85,33 @@ func makeDownloadAPI(
 	activeRemoteRequests *types.ActiveRemoteRequests,
 	activeThumbnailGeneration *types.ActiveThumbnailGeneration,
 ) http.HandlerFunc {
-	return prometheus.InstrumentHandler(name, http.HandlerFunc(func(w http.ResponseWriter, req *http.Request) {
-		req = util.RequestWithLogging(req)
+	return promhttp.InstrumentHandlerCounter(
```

Comment:
This is a bit more difficult to read than it should be, can't you define the `CounterVec` and the `HandlerFunc` separately and then do `promhttp.InstrumentHandlerCounter(counterVec, handlerFunc)` or something like that? Or just defining separately the function given to `http.HandlerFunc` like you do in `common/httpapi.go` is also good enough.